app/Main.hs

```diff
@@ -169,22 +145,24 @@ parseValidate Config{..} = do
     d''' <- case confOutputInput of
          "" -> return d''
          x  ->  return $ progMirrorInputRelations d'' x
+    when confDumpFlat $
+        writeFile (replaceExtension confDatalogFile ".flat.ast") (show d''')
```

Comment:
The ast suffix does not really make sense. I think you should use the dl suffix, then syntax highlighting tools will work properly.

Reply:
I thought about this, but unfortunately this is not valid DDlog code, as it includes flattened names containing dots.